app/views/finders/_show_header.html.erb

```diff
@@ -66,6 +66,10 @@
       <% end %>
     </div>
 
+    <% content_for :meta_tags do %>
+      <%= @requested_variant.analytics_meta_tag.html_safe %>
```

Comment:
if you do add a `page_under_test?` method, you'd need to wrap this bit in it too